plugins/formal-verification/formver.plugin/src/org/jetbrains/kotlin/formver/FormalVerificationPluginErrorMessages.kt

```diff
@@ -37,5 +38,11 @@ object FormalVerificationPluginErrorMessages : BaseDiagnosticRendererFactory() {
             "Function may return a {0} value.",
             CommonRenderers.STRING
         )
+        put(
+            PluginErrors.INVALID_INVOCATION_TYPE,
+            "Could not verify that function `{0}` is called {1}.",
```

Comment:
Is the use of ` standard for Kotlin errors?  I thought ' was the norm.

Reply:
Yes, I'm going to correct that. Thank you for the feedback.